neurons/validators/validator.py

```diff
@@ -206,6 +206,38 @@ async def get_uids(
         # uid_list = list(available_uids.keys())
         return uids.to(self.config.neuron.device)
 
+    async def get_uids_from_list(
+        self,
+        uids,
+        strategy=QUERY_MINERS.RANDOM,
+        is_only_allowed_miner=False,
+        specified_uids=None,
+    ):
```

Comment:
change old get_uids instead of adding new duplicated

Reply:
done